tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -467,19 +467,18 @@ class MklConvOp : public OpKernel {
                                         filter.shape().DebugString()));
 
     for (int i = 0; i < 3; i++) {
```

Comment:
I know you didn't do this but could it change this ```i++``` to ```++i``` while at it, please..

Reply:
Done. Thanks!